definitions/divorce/json/CaseEvent/CaseEvent-general-referral-nonprod.json

```diff
@@ -24,5 +24,17 @@
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/general-consideration",
     "SecurityClassification": "Public",
     "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "generalReferralPayment",
+    "Name": "General Referral Payment",
+    "Description": "General Referral Payment",
+    "DisplayOrder": 18,
+    "PreConditionState(s)": "AwaitingGeneralReferralPayment",
+    "PostConditionState": "AwaitingGeneralConsideration",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/set-up-confirm-service-payment",
```

Comment:
I think we should rename this endpoint.
Technically what it is doing is more something like: 'set-up-order-summary/without-notice-fee'

Reply:
Id be happy to change that. Other than that, is the rest of the PR suitable?

Reply:
updated